### PR TITLE
Health bar

### PR DESCRIPTION
**Pull request by x183**
Should now render properly

### CODE REVIEW

File: core/src/main/java/com/g16/feyrune/model/combat/CombatModel.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public CombatModel(Player player, Encounter encounter) {
     }
 
     public void fillCombatCreatureList(Player player, Encounter encounter) {
-        combatCreatures.add(new PlayerCreature(CreatureFactory.createCreature())); // FIX: Should be player.getMonster()
+        combatCreatures.add(new PlayerCreature(player.getCreature())); // FIX: Should be player.getMonster()
```
Comment (lutches) [inline core/src/main/java/com/g16/feyrune/model/combat/CombatModel.java:32]: It feels like the //FIX has been implemented? maybe remove it?

File: core/src/main/java/com/g16/feyrune/model/player/Player.java [hunk 2]
```diff
@@ -20,6 +22,7 @@ public Player(String name, Point coordinates) {
         this.coordinates = coordinates;
         texture = new Texture("assets/hero/humanmale/humanMale.png");
         this.observers = new ArrayList<>();
+        creature = CreatureFactory.createCreature();
```
Comment (lutches) [inline core/src/main/java/com/g16/feyrune/model/player/Player.java:25]: should the Player have a connection to creature factory?

Comment (x183) [reply core/src/main/java/com/g16/feyrune/model/player/Player.java:25]: It's a temporary fix until we implement a way for the players to acquire new creatures 🙂

**Review by lutches — APPROVED** [review]
left a few comments but it looks fine

**Pull request merged**